dspace-api/src/main/java/org/dspace/util/SolrUpgradeStatistics6.java

```diff
@@ -109,9 +195,17 @@ public static void main(String[] args) throws ParseException
         {
                 CommandLineParser parser = new PosixParser();
                 Options options = makeOptions();
+                
+                System.out.println(" * This process should be run iteratively over every statistics shard ");
+                System.out.println(" * until there are no remaining records with legacy ids present.");
+                System.out.println(" * This process can be run while the system is in use.");
+                System.out.println(" * It possible, run this tool with -Xmx2000m .");
```

Comment:
Yes, looks to be a typo. We also may wish to reword this slightly to add more info as to what "-Xmx" means, if it isn't clear to some users:
"If possible, please allocate 2GB of memory to this process (e.g. -Xmx2000m)"